docs/src/developers_guide/contributing_ci_tests.rst

```diff
@@ -72,14 +72,11 @@ New lockfiles are generated automatically each week to ensure that Iris continue
 tested against the latest available version of its dependencies.
 Each week the yaml files in ``requirements/ci`` are resolved by a GitHub Action.
 If the resolved environment has changed, a pull request is created with the new lock files.
-The CI test suite will run on this pull request and fixes for failed tests can be pushed to
-the ``auto-update-lockfiles`` branch to be included in the PR. 
-Once a developer has pushed to this branch, the auto-update process will not run again until
-the PR is merged, to prevent overwriting developer commits.
-The auto-updater can still be invoked manually in this situation by going to the `GitHub Actions`_
-page for the workflow, and manually running using the "Run Workflow" button.  
-By default, this will also not override developer commits.  To force an update, you must 
-confirm "yes" in the "Run Worflow" prompt.
+The CI test suite will run on this pull request. If the tests fail, a developer
+will need to create a new branch based off the ``auto-update-lockfiles`` branch 
+and add the required fixes to this new branch. If the fixes are made to the
```

Comment:
Could we add something to this effect in the body of the PR that gets raised during the Action's `create_pr` job?

Reply:
I added this in the latest commit (https://github.com/SciTools/iris/pull/4618/commits/ccf4a5532a6d886525999ea33cec55be5accf7a0) and when I tested it, it appeared to work:
![image](https://user-images.githubusercontent.com/8101163/157237569-560a0a4f-f078-45d8-b087-d7196422d533.png)